### PR TITLE
Colombia gov

### PR DESCRIPTION
**Pull request by jsebastianms1**
Added data from Colombia government 

Adding functionality for  issue #740

### CODE REVIEW

**Comment by jsebastianms1** [comment]
> The tests are wrong. You are calling Vietnam endpoint in the test.

I will fix this

File: scrapers/govScrapers/getColombia.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const axios = require('axios');
+const logger = require('../../utils/logger');
+
+const filters = ['Recuperado', 'Fallecido'];
+
```
Comment (ebwinters) [inline scrapers/govScrapers/getColombia.js:5]: delete extra newline

File: scrapers/govScrapers/getColombia.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+const transformData = (data, cities) => {
+	const state = 13;
+	const pos = cities
+		? 11 : 12;
```
Comment (ebwinters) [inline scrapers/govScrapers/getColombia.js:10]: put this on line 9 for better readability

File: scrapers/govScrapers/getColombia.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+const axios = require('axios');
+const logger = require('../../utils/logger');
+
+const filters = ['Recuperado', 'Fallecido'];
```
Comment (ebwinters) [inline scrapers/govScrapers/getColombia.js:4]: Why do you have this variable in an Array? Just use the strings as keys on lines 11&12

File: scrapers/govScrapers/getColombia.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+			return acc;
+		}, {});
+
+/**
+ * Scrapes Colombian government javascript data and returns an object containing this data
+ */
+const colombiaData = async () => {
+	try {
+		const colombiaResponse = (await axios.get('https://www.datos.gov.co/api/views/gt2j-8ykr/rows.json')).data;
+		console.log(colombiaResponse.data);
```
Comment (ebwinters) [inline scrapers/govScrapers/getColombia.js:41]: Remove `console.log` and use our logger. Instead log something like `logger.info('Successfuly scraped xyz');`

File: tests/api_gov.spec.js [hunk 5]
```diff
@@ -344,5 +344,21 @@ describe('TESTING /v2/gov/new zealand', () => {
 				});
 				done();
 			});
-	});
+  });
+
+  describe('TESTING /v2/gov/colombia', () => {
+    it('/v2/gov/vietnam correct fields set', (done) => {
```
Comment (ebwinters) [inline tests/api_gov.spec.js:350]: Fix copying mistake

File: tests/api_gov.spec.js [hunk 6]
```diff
@@ -344,5 +344,21 @@ describe('TESTING /v2/gov/new zealand', () => {
...
+      chai.request(app)
+        .get('/v2/gov/vietnam')
```
Comment (ebwinters) [inline tests/api_gov.spec.js:352]: wrong endpt

**Comment by jsebastianms1** [comment]
> Please see my comments and fix. Thanks for the contribution!

> The tests are wrong. You are calling Vietnam endpoint in the test.

Hello, can you please check again? thanks to both of you

File: scrapers/govScrapers/getColombia.js (hunk 4 above)
Comment (jsebastianms1) [reply scrapers/govScrapers/getColombia.js]: removed

**Comment by EliteDaMyth** [comment]
Looks good. Merging.

**Comment by EliteDaMyth** [comment]
@all-contributors please add @jsebastianms1 for code

**Comment by EliteDaMyth** [comment]
https://disease.sh/v2/gov/colombia

**Pull request merged**